Makefile

```diff
@@ -74,14 +74,16 @@ curl_local curl_prod:
 	--data "@./data/sample_deal.json" \
 	${CURL_URL}
 
-logfile := logs/src.api.$(shell date +%FT%T).log
-pidfile := logs/server.PID
+LOGDIR  := logs/
+logfile := ${LOGDIR}src.api.$(shell date +%FT%T).log
+pidfile := ${LOGDIR}server.PID
```

Comment:
For me, it would be a bit easier to read if LOGDIR didn't have the last slash. It would make it explicit that pidfile and logfile variables are inside a directory when reading the code quickly.

Reply:
Done.